tsconfig.json

```diff
@@ -3,7 +3,7 @@
 		"target": "ES2020",
 		"useDefineForClassFields": true,
 		"lib": ["ES2020", "DOM", "DOM.Iterable"],
-		"module": "ESNext",
```

Comment:
import.meta will break with es2020

requesting change to esnext again

Reply:
Actually I think we should skip unit test for  components of big brain and rather focus on integration tests